.rubocop.yml

```diff
@@ -33,6 +33,10 @@ Layout/EmptyLineAfterGuardClause:
 Layout/LineLength:
   Max: 150
 
+Lint/MissingSuper:
+  Exclude:
+    - 'app/components/**/*'
+
```

Comment:
ViewComponents don't need to call `super`. We don't do it in prescat or H2 or anywhere, so rubocop needn't care about this rule for components.

Reply:
👍🏻  rubocop is a jerk, sometimes.